src/main/java/uk/gov/hmcts/juror/api/bureau/service/JurorCommsSentToCourtServiceImpl.java

```diff
@@ -70,105 +60,140 @@ public SchedulerServiceClient.Result process() {
                 }
 
                 //update regardless - stop processing next time.
                 jurorDetail.getJuror().setNotifications(ALL_NOTIFICATION_SENT);
                 notificationsSent = ALL_NOTIFICATION_SENT;
                 update(jurorDetail);
+                successCount++;
 
             } catch (JurorCommsNotificationServiceException e) {
-                log.error(
-                    "Unable to send sent to court comms for {} : {} {}",
-                    jurorDetail.getJurorNumber(),
-                    e.getMessage(),
-                    e.getCause().toString()
-                );
-                errorCount++;
+                boolean isError = false;
+                if (isEmail) {
+                    if (NotifyUtil.isInvalidEmailAddressError(e.getCause())) {
+                        errorInvalidEmailCount++;
+                    } else {
+                        isError = true;
+                        errorCountEmail++;
+                    }
+                }
+                if (isSms) {
+                    if (NotifyUtil.isInvalidPhoneNumberError(e.getCause())) {
+                        errorInvalidPhoneCount++;
+                    } else {
+                        isError = true;
+                        errorCountSms++;
+                    }
+                }
+                if (isError) {
+                    errorCount++;
+                    log.error(
+                        "Unable to send sent to court comms for {}",
+                        jurorDetail.getJurorNumber(), e
+                    );
+                }
                 if (notificationsSent.equals(EMAIL_NOTIFICATION_SENT)) {
                     jurorDetail.getJuror().setNotifications(notificationsSent);
                     update(jurorDetail);
                 }
             } catch (Exception e) {
-                log.error("Sent To Court Comms Processing : Juror Comms failed : {}", e.getMessage());
+                log.error("Sent To Court Comms Processing : Juror Comms failed : {}", e.getMessage(), e);
                 errorCount++;
+                if (isEmail) {
+                    errorCountEmail++;
+                }
+                if (isSms) {
+                    errorCountSms++;
+                }
             }
         }
         log.info("Sent To Court Comms Processing : Finished - {}", dateFormat.format(new Date()));
         return new SchedulerServiceClient.Result(
             errorCount == 0
                 ? SchedulerServiceClient.Result.Status.SUCCESS
                 : SchedulerServiceClient.Result.Status.PARTIAL_SUCCESS, null,
-            Map.of("ERROR_COUNT", "" + errorCount));
+            Map.of(
+                "SUCCESS_COUNT_EMAIL", String.valueOf(successCountEmail),
+                "SUCCESS_COUNT_SMS", String.valueOf(successCountSms),
+                "ERROR_COUNT_EMAIL", String.valueOf(errorCountEmail),
+                "ERROR_COUNT_SMS", String.valueOf(errorCountSms),
+
```

Comment:
do we need this line break?  and the one after this block

Reply:
Updated :)